### DISCUSSION TITLE
Please add support for standard macOS centre in addition to screen centre

### DISCUSSION THREAD

**Discussion by deandmx** [rxhanson/RectanglePro-Community#120] [Ideas]
The traditional standard screen centre for macOS has always been not screen centre, but more like ⅔ up. You'll notice that when you launch an installer or another stock app and you apply the 'centre' command, it shifts down to centre screen. That starting position is macOS centre. 

For me, macOS centre just feels more comfortably eye level. I'd love to see support for it. Other window management apps like Many Tricks' [Moom](https://manytricks.com/moom/) have this and it's a nice detail that shows their knowledge of Mac history.

<img width="1512" alt="Screenshot 2022-09-23 at 12 22 09 pm" src="https://user-images.githubusercontent.com/49750/191951131-e72144c5-0d7a-48f3-880c-923c0b249e3d.png">

<img width="1512" alt="Screenshot 2022-09-23 at 12 23 12 pm" src="https://user-images.githubusercontent.com/49750/191951155-f21363da-c4ba-4c21-a534-4833e276d836.png">

**Comment by rxhanson** [comment]
Sure, I can add this in an upcoming release.

**Comment by deandmx** [comment]
Top man 🤝
